eng/Publishing.props

```diff
@@ -1,11 +1,12 @@
-<Project InitialTargets="ValidateDownloadedAssets">
-
+<Project>
   <PropertyGroup>
     <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
     <!-- This avoids creating VS.*.symbols.nupkg packages that are identical to the original package. -->
     <AutoGenerateSymbolPackages>false</AutoGenerateSymbolPackages>
-    <!-- Set PlatformName to TargetArchitecture to create unique build manifest files. -->
-    <PlatformName Condition="'$(TargetArchitecture)' != ''">$(TargetArchitecture)</PlatformName>
+    <!-- Except for jobs that set a specific asset manifest file name as a global property, set the manifest name based on the package RID. -->
+    <AssetManifestFileName Condition="'$(AGENT_JOBNAME)' != ''">$(AGENT_JOBNAME)</AssetManifestFileName>
+    <!-- Reset the file path now that we may have changed the name. -->
+    <AssetManifestFilePath>$(ArtifactsLogDir)AssetManifest\$(AssetManifestFileName)</AssetManifestFilePath>
```

Comment:
You can revert this after https://github.com/dotnet/runtime/pull/112013 is merged:

```suggestion
```